### ISSUE TITLE
loss function class is not preserved in save/load of model

### ISSUE THREAD

**Issue by sammlapp** [kitzeslab/opensoundscape#283]
If the user changes the loss function of PytorchModel child class, then saves using .save(), then loads and tries to continue re-training, the change of loss function will not be recovered. This isn't as easy to address as other saved changes because we need to point to a class.

**Comment by sammlapp** [comment]
I no longer see this as a priority, because swapping out various loss functions is not a casual activity to be done on the fly. The loss function should match the network and sampling approaches, and chaining a loss function will often involve writing a new class that overrides the _init_loss_fn() method. Perhaps the user should not even be allowed to change model.loss_cls, because it will rarely turn out well.
